src/main/fc/config.c

```diff
@@ -631,6 +631,10 @@ void validateAndFixConfig(void)
     validateAndFixGyroConfig();
 #endif
 
+#ifdef RUNCAM_SPLIT_SUPPORT
+    featureSet(FEATURE_RUNCAM_SPLIT);
```

Comment:
Feature is probably not necessary - it can be enabled just by assigning FUNCTION_RUNCAM_SPLIT_CONTROL to serial port. Also Runcam is not that common (I don't have one yet ;-) ) to enable it by default